examples/pmpong/CMakeLists.txt

```diff
@@ -57,3 +57,23 @@ add_executable(pmpong Ball.cpp GameController.cpp GameOverView.cpp
 		PongGameStatus.cpp Pool.cpp)
 target_include_directories(pmpong PUBLIC ${SFML_INCLUDE_DIR} ${LIBPMEMOBJ++_INCLUDE_DIRS} .)
 target_link_libraries(pmpong ${LIBPMEMOBJ++_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT} ${SFML_LIBRARIES})
+
+if(NOT WIN32)
+	execute_process(COMMAND uname -s OUTPUT_VARIABLE SYSTEM_TYPE)
```

Comment:
Fonts path is not kernel-specific, it's userland-specific, eg this will fail on Debian GNU/kFreeBSD.

I think the only sane option is to use fontconfig's fc-list tool.